src/Microsoft.Identity.Web/Constants/Constants.cs

```diff
@@ -68,5 +68,9 @@ internal static class Constants
         // Microsoft Graph
         public const string UserReadScope = "user.read";
         public const string GraphBaseUrlV1 = "https://graph.microsoft.com/v1.0";
+
+        // Telemetry headers
+        public const string TelemetryHeaderKey = "x-client-brkrver";
+        public const string IDWebSku = "IDWeb.";
```

Comment:
`IDWebSku` -> `TelemetryIdWebSku`?
`"IDWeb."` -> `IdWeb.`?

Reply:
not sure what you mean...for the top one as it's not telemetry specific, it's just our sku value, for the 2nd one, this is to have a `.` between the sku and the version in kusto. see the example above.

---
In reply to: [487377300](https://github.com/AzureAD/microsoft-identity-web/pull/572#discussion_r487377300) [](ancestors = 487377300)